package.json

```diff
@@ -12,18 +12,16 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint src/components"
+    "lint": "eslint --ext .js,.jsx src/client",
```

Comment:
I think you can `eslint xxx && prettier yyy` (xxx and yyy are your arguments for eslint and prettier) so that you can lint for code smell and coding styles via both eslint and prettier.